Makefile-tests.am

```diff
@@ -33,10 +33,6 @@ TESTS_ENVIRONMENT += OT_TESTS_DEBUG=1 \
 	PATH=$$(cd $(top_builddir)/tests && pwd):$${PATH} \
 	$(NULL)
 
-uninstalled_test_data = tests/ostree-symlink-stamp
-
-dist_uninstalled_test_scripts = tests/test-symbols.sh
-
```

Comment:
This commit got screwed up. It looks like Simon McVittie fixed a lot of these dist problems upstream, so I think this commit can be dropped.